cmd/helmrelease.go

```diff
@@ -185,6 +187,15 @@ with kubectl.`,
 	},
 }
 
+func GetHelmRepoNamespace(helmRelease *helmv2.HelmRelease) string {
+	helmRepoNamespace := helmRelease.Spec.Chart.Spec.SourceRef.Namespace
+	if helmRepoNamespace == "" {
+		helmRepoNamespace = "default"
```

Comment:
I'm trying to think of a better default than `default`. Ideally we should default to the namespace of the helmRelease

```
if  helmRepoNamespace == "" {
        helmRepoNamespace = helmRelease.Namespace
}
```

Or maybe just `flux-system` (as that's the default for most flux users).

Reply:
The users will be coming from the Flux background, and according to the [Flux documentation](https://fluxcd.io/flux/components/source/helmrepositories/#helm-https-repository) the default value for the namespace is default. I think we should stick to default rather than any other namespace.

Reply:
cc @hiddeco